FirebaseAuthExchange/Apps/AuthExchange/Podfile

```diff
@@ -5,7 +5,7 @@ source 'https://github.com/firebase/SpecsStaging.git'
 source 'https://cdn.cocoapods.org/'
 
 def shared_pods
-  pod 'FirebaseCore' 
+  pod 'FirebaseCore'
   pod 'FirebaseInstallations'
```

Comment:
Paths should be added for all Firebase products to test with the versions in the repo instead of released versions.